src/lib/Middleware.js

```diff
@@ -0,0 +1,61 @@
+// We are not using NextJS's native middleware featuer
+// because Amplify might not support it.
+import multer from "multer";
+import { verifyToken } from "../lib/JWT";
+import InvalidTokenError from "../errors/InvalidTokenError";
+import MissingHeaderError from "../errors/MissingHeaderError";
+
+// Uncomment when KlaudSol CMS is in v2.0.0
+/* export const parseFormData = async (req, res) => { */
+/*   if (req.method !== "POST" && req.method !== "PUT") return; */
+/**/
+/*   const storage = multer.memoryStorage(); */
+/*   const multerSetup = multer({ storage }); */
+/*   const upload = multerSetup.any(); */
+/**/
+/*   await new Promise((resolve, reject) => { */
+/*     upload(req, res, (result) => { */
+/*       if (result instanceof Error) return reject(result); */
+/*       return resolve(result); */
+/*     }); */
+/*   }); */
+/**/
+/*   req.body = JSON.parse(JSON.stringify(req.body)); */
+/* }; */
+
+const tokenExtractor = async (req, res) => {
+  req.token = req.token.substring(7);
```

Comment:
Can we use a constant here instead of a magic number?
```suggestion
  req.token = req.token.substring(7);
```